app/public/fonts/bootstrap/glyphicons-halflings-regular.svg

```diff
@@ -0,0 +1,288 @@
+<?xml version="1.0" standalone="no"?>
```

Comment:
Why did you change the bootstrap font graphic files in this commit?  That seems unrelated to the password change issue.  I noticed you had these files changed in your practice pull request too.   Was this a mistake?

Reply:
Actually, whenever I work with the new branch it doesn't have public/fonts folder so it creates them. once it is merged with the master it won't happen again.

Reply:
gulp generates those files, though.  Tracking them is like tracking the app.css file that all of our scss is converted into.

If you're bothered by running gulp when changing branches, you could either keep the files untracked/uncommitted too.  You may also like using 'gulp watch' instead of just 'gulp' since 'gulp watch' will automatically run when changed are saved to input files.

I merged a change tonight to ignore files in public/fonts/bootstrap to help with the issue you ran into and clarify why we don't want to track those files.

Please delete your public/fonts/bootstrap directory and squash or fixup that into your other commit.  Combining the changes into a single commit will prevent a conflict between other developers having those files untracked and one of your commits adding the same files.